src/PowerShell/Microsoft.WinGet.Client.Engine/Helpers/AppxModuleHelper.cs

```diff
@@ -73,12 +73,12 @@ internal class AppxModuleHelper
         private const string VCLibsUWPDesktopArm64 = "https://aka.ms/Microsoft.VCLibs.arm64.14.00.Desktop.appx";
 
         // Xaml
-        private const string XamlPackage27 = "Microsoft.UI.Xaml.2.7";

```

Comment:
Just swapping to 2.8 will mean that older versions won't install.  I would rather have a fix that handles both situations.